scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -33,8 +33,8 @@
 
 # We always run the following code in the kernel before running the notebook
 PRE_EXECUTE_CODE = """\
-import warnings
-warnings.filterwarnings("ignore", message="Matplotlib is building the font cache; this may take a moment.")
+import matplotlib
+matplotlib.set_loglevel("critical")
```

Comment:
Can you please link to the GitHub issue for why we set this?

Reply:
```suggestion
# See https://github.com/matplotlib/matplotlib/issues/23326#issuecomment-1164772708
matplotlib.set_loglevel("critical")
```